src/client/src/models/Scene.js

```diff
@@ -108,22 +108,47 @@ const Scene = {
     Scene.setScenes(list)
   },
 
+
+
+
+
+
+
   // this function will save the current scene to the localStorage
   saveStart: function() {
     const list = Scene.getScenes()
 
     // this is a check to make sure the value entered into start is a number
     const start_check = Number(Scene.currentScene.start)
     if (isNaN(start_check)) {
-      Scene.currentScene.start = ``
+      Scene.current.start = null;
+    } else {
+      var i;
+      for (i = 0; i < list.length; i++) {
+        if (list[i].start === null) {
+          //do nothing
+        } else if (Number(list[i].start) === start_check && i !== Scene.current.id - 1) {
+          Scene.current.start = `Collision Detected`
+        }
+      }
     }
-
     // update the current scenes start in the scene list
     list[Scene.findSceneIndex(Scene.currentScene.id)].start = Scene.currentScene.start
-
+    list.sort(function(a, b){
+      if(b.start === null) {return -1} // undefined starts are sorted at the end
```

Comment:
This actually causes a weird bug where all starts with null switch places when you save changes. You should add an extra if statement at the beginning that looks like this.
`if (b.start === null && a.start === null) {return 0}`
This will make it so that if the two are both null, they don't switch places.

Reply:
Done and solved.